dev-docs/bidder-adapter-1.md

```diff
@@ -358,6 +357,39 @@ function createBid(status, reqBid, response) {
 
 {% endhighlight %}
 
+#### If you would like to participate in auctions for outstream video ads
+
+Prebid.js passes a signal on video impressions that indicates whether the video context is instream or outstream.  You can start ingesting this signal if you wish.
+
+As described in [Show Outstream Video Ads]({{site.baseurl}}/dev-docs/show-outstream-video-ads.html), video ad units have a publisher-defined video context, which can be either `"instream"` or `"outstream"`:
```

Comment:
Let's combine this paragraph with the one above into something like

"As described in...video ad units may define a video context of 'instream' or 'outstream'.  Video demand partners can choose to ingest this signal for targeting purposes.